freedomOfComputing/src/main/java/nl/tudelft/trustchain/FOC/community/FOCCommunity.kt

```diff
@@ -139,7 +139,6 @@ class FOCCommunity(
         val torrentHash = payload.message.substringAfter("magnet:?xt=urn:btih:")
             .substringBefore("&dn=")
         if (torrentMessagesList.none {
-            it.second
```

Comment:
this looked like a typo in a previous PR, I've removed it, wasn't sure if it was appropriate to do it as a part of this PR - will remove it if you'd rather I commit it separately